src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -79,3 +161,982 @@ def chamfer(
+        extrude_type : ExtrudeType, default: ExtrudeType.ADD
+            Type of extrusion to be performed.
+        offset_mode : OffsetMode, default: OffsetMode.MOVE_FACES_TOGETHER
+            Mode of how to handle offset relationships.
+        pull_symmetric : bool, default: False
+            Pull symmetrically on both sides if ``True``.
+        copy : bool, default: False
+            Copy the face and move it instead of extruding the original face if ``True``.
+        force_do_as_extrude : bool, default: False
+            Forces to do as an extrusion if ``True``, if ``False`` allows extrusion by offset.
+
+        Returns
+        -------
+        List[Body]
+            Bodies created by the extrusion if any.
+        """
+        from ansys.geometry.core.designer.face import Face
+
+        faces: list[Face] = faces if isinstance(faces, list) else [faces]
+        check_type_all_elements_in_iterable(faces, Face)
+
+        for face in faces:
+            face.body._reset_tessellation_cache()
+
+        result = self._commands_stub.ExtrudeFacesUpTo(
+            ExtrudeFacesUpToRequest(
+                faces=[face._grpc_id for face in faces],
+                up_to_selection=up_to_selection._grpc_id,
+                seed_point=point3d_to_grpc_point(seed_point),
+                direction=unit_vector_to_grpc_direction(direction),
+                extrude_type=extrude_type.value,
+                pull_symmetric=pull_symmetric,
+                offset_mode=offset_mode.value,
+                copy=copy,
+                force_do_as_extrude=force_do_as_extrude,
+            )
+        )
+
+        design = get_design_from_face(faces[0])
+
+        if result.success:
+            bodies_ids = [created_body.id for created_body in result.created_bodies]
+            design._update_design_inplace()
+            return get_bodies_from_ids(design, bodies_ids)
+        else:
+            self._grpc_client.log.info("Failed to extrude faces.")
+            return []
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def extrude_edges(
+        self,
+        edges: Union["Edge", List["Edge"]],
+        distance: Real,
+        from_face: "Face" = None,
+        from_point: Point3D = None,
+        direction: UnitVector3D = None,
+        extrude_type: ExtrudeType = ExtrudeType.ADD,
+        pull_symmetric: bool = False,
+        copy: bool = False,
+        natural_extension: bool = False,
+    ) -> List["Body"]:
+        """Extrude a selection of edges. Provide either a face or a direction and point.
+
+        Parameters
+        ----------
+        edges : Edge | List[Edge]
+            Edges to extrude.
+        distance : Real
+            Distance to extrude.
+        from_face : Face, default: None
+            Face to pull normal from.
+        from_point : Point3D, default: None
+            Point to pull from. Must be used with ``direction``.
```

Comment:
```suggestion
            Point to pull from. This parameter must be used with the ``direction`` parameter.
```
Complete sentences should be used after the brief description.